docs/ui/components/Step/Step.tsx

```diff
@@ -0,0 +1,66 @@
+import { css } from '@emotion/react';
+import { spacing, theme, typography } from '@expo/styleguide';
+import { PropsWithChildren } from 'react';
+
+type Props = PropsWithChildren<{
+  label: string;
+  connected?: boolean;
+}>;
+
+export const Step = ({ children, label, connected }: Props) => {
+  return (
+    <div css={stepWrapperStyle}>
+      <div>
+        <div css={stepLabelStyle}>{label}</div>
+        {connected && <div css={stepConnectionStyle} />}
+      </div>
+      <div css={stepContentStyle}>{children}</div>
+    </div>
+  );
+};
+
+const stepWrapperStyle = css({
+  display: 'grid',
+  gridTemplateColumns: `${spacing[9]}px minmax(0, 1fr)`,
+  gap: spacing[4],
+  margin: `${spacing[2]}px 0`,
+  float: 'left',
+  width: '100%',
+});
+
+const stepLabelStyle = css({
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  alignSelf: 'center',
+  margin: `0 auto`,
+  width: spacing[9],
+  height: spacing[9],
+  fontFamily: typography.fontStacks.semiBold,
+  color: theme.text.secondary,
```

Comment:
I will leave it up to you, but once we darken the background color a shade, I bet this will look better as `theme.text.default`. If you like secondary more, add the `theme="secondary"` prop to the `<HEADLINE>` element.